README.md

```diff
@@ -1,29 +1,78 @@
 # Intro to REST
 
-## Objectives
+## What is REST?
 
-1. Define the RESTful URL conventions
-2. Identify seven RESTful URLs and HTTP verbs
-3. Identify seven RESTful actions
+REST stands for REpresentational State Transfer, essentially REST is a standardized design architecture that enables applications to communicate and manage data flow. REST was originally proposed in 2000 by Roy Thomas Fielding in his PhD dissertation as an architectural style for connecting the Internet. At that time the Internet was growing fast, however there was a distinct lack of organization when it came to web applications communicating with each other. RESTful principles helped to bridge the gap between applications and aided in laying the foundation for the Internet ecosystem that exists today.
```

Comment:
This section is super super jargon-y. I would rephrase this to be:

"In 2000, Roy Thomas Fielding was frustrated by the hap-hazard different ways web applications were using the HTTP standard. Specifically he was frustrated with how URLs and their corresponding HTTP Verbs were used differently for every single application. So, in his PhD dissertation he came up with REST or REpresentational State Transfer as a standard way web apps should structure their URLs. It also suggested a few other things, but we focus mostly on how it changed URLs. Fielding, also noticed the rise in web applications communicating with each other. Using this standard way of forming URLs to access resources, Fielding hoped that inter-application communication would get much easier."